run.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+import sys
+from os.path import join, dirname, realpath
+
+base_path = dirname(realpath(__file__))
+source_path = join(base_path, "src", "main", "python")
+script_path = join(base_path, "src", "main", "scripts", "cbas")
+
+sys.path.insert(0, source_path)
+exec(open(script_path, 'r').read())
```

Comment:
I'll merge this now, but would you consider adding this as a task to the `build.py`.